public/scripts/ebayCall.js

```diff
@@ -0,0 +1,65 @@
+// Parse the response and build an HTML table to display search results
+// function _cb_findItemsByKeywords(root) {
+function _cb_findItemsAdvanced  (root) {
+  var items = root.findItemsAdvancedResponse[0].searchResult[0].item || [];
+  var html = [];
+  html.push('<table width="75%" border="0" cellspacing="0" cellpadding="3" id="ebayTable"><tbody>');
```

Comment:
I know Laura really wanted to refactor how she was using the Ebay API and associated code snippets... I see what she was cringing at now! :-)

There's a lot of work that could be done to improve this code. It might be a great project, which you can complete and then reach out to Ebay directly to see if you can provide some updates to their API docs!